partner_preview_scripts/ot_crispr.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env python3
```

Comment:
Same as in the previous parser, I believe a short description of the overall process (about one short paragraph's worth) would be quite benefitial. Essentially what you wrote in the PR description, but added to the module's docstring